### ISSUE TITLE
[1.14.4] Addon doesn't show bags and gives out Lua errors

### ISSUE THREAD

**Issue by nezorflame** [Nevcairiel/Inventorian#32]
Hi! I'm having multiple errors on Hardcore Era.

1. Multiple `SharedXML` errors:

	<details>
	<summary>Details</summary>
	
	```lua
	18x SharedXML/SharedUIPanelTemplates.xml:1474 Texture Texture: Unable to load atlas entry UI-Frame-DiamondMetal-Header-CornerRight
	```
	
	</details>

2. `CURSOR_UPDATE` error:

	<details>
	<summary>Details</summary>
	
	```lua
	1x InventorianBag2:RegisterEvent(): InventorianBag2:RegisterEvent(): Attempt to register unknown event "CURSOR_UPDATE"
	[string "=[C]"]: in function `RegisterEvent'
	[string "@Inventorian/Bag.lua"]:88: in function `Set'
	[string "@Inventorian/Frame.lua"]:214: in function `UpdateBags'
	[string "@Inventorian/Frame.lua"]:202: in function `ToggleBagFrame'
	[string "@Inventorian/Frame.lua"]:127: in function `OnBagToggleClick'
	[string "*Frame.xml:242_OnClick"]:1: in function <[string "*Frame.xml:242_OnClick"]:1>
	
	Locals:
	(*temporary) = InventorianBag2 {
	 0 = <userdata>
	 hasItem = "|cffffffff|Hitem:4496::::::::14:::::::::|h[Small Brown Pouch]|h|r"
	 count = InventorianBag2Count {
	 }
	}
	(*temporary) = "CURSOR_UPDATE"
	```
	
	</details>

@Nevcairiel any chance you could look into this?

**Comment by nezorflame** [comment]
Figured out the fix - 1.14.4 version doesn't have `CURSOR_UPDATE` anymore, so you need to delete one `if` block.
Gonna make a PR soon.

**Comment by nezorflame** [comment]
@Nevcairiel please take a look at PR
